### PR TITLE
switch to pdf2pdfocr

### PR DESCRIPTION
**Pull request by gabriel-v**
Pretty output and user language selection (through directory name).

### CODE REVIEW

File: batch.py [hunk 1]
```diff
@@ -26,16 +25,35 @@
...
 def walk_files():
-    for root, _, files in os.walk(os.path.abspath(DATA_DIR)):
-        for filename in files:
-            full_path = os.path.join(root, filename)
-            if os.path.splitext(full_path)[1] in ALL_EXTENSIONS:
-                yield full_path
+    for language in os.listdir(DATA_DIR):
+        language_path = os.path.abspath(os.path.join(DATA_DIR, language))
+        if not os.path.isdir(language_path):
+            log.warning("%s is not a directory", language_path)
```
Comment (mgax) [inline batch.py:45]: Do we really want this log spam?

File: batch.py [hunk 2]
```diff
@@ -26,16 +25,35 @@
...
+            continue
+        if language not in ALL_LANGUAGES:
+            log.warning("%s is not a valid language", language)
```
Comment (mgax) [inline batch.py:48]: Log spam here too

File: batch.py [hunk 3]
```diff
@@ -78,16 +95,20 @@ def run_tesseract(image_path, output_stem):
 
 
 def main():
+    log.info('copying example files...')
+    subprocess.check_call(['cp', '-a', '/examples/.', DATA_DIR])
```
Comment (mgax) [inline batch.py:99]: This feels impolite, let's not spam our users

**Review by mgax — CHANGES_REQUESTED** [review]

File: batch.py (hunk 1 above)
Comment (gabriel-v) [reply batch.py:45]: spammy anyway

**Review by mgax — APPROVED** [review]

**Pull request merged**